UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -534,11 +590,19 @@
       return '<button class="btn edit-only editable btn-secondary add-panel" id="'+id+'" >Uusi lisäkilpi</button>';
     };
 
-    var singleChoiceForPanelTypes = function (property, collection) {
+    var singleChoiceForPanelTypes = function (property, collection, asset) {
       var propertyValue = _.isUndefined(_.last(property))  ? '' : _.last(property);
       var signTypes = _.map(_.filter(me.enumeratedPropertyValues, function(enumerated) { return enumerated.publicId == 'trafficSigns_type' ; }), function(val) {return val.values; });
       var panels = _.find(collection.getAdditionalPanels(signTypes));
-      var propertyDisplayValue = _.find(panels, function(panel){return panel.propertyValue == propertyValue.toString();}).propertyDisplayValue;
+      var propertyDisplayValue;
+
+      if (isPedestrianOrCyclingRoadLink(asset)) {
+        panels = _.filter(panels, function(p) { if (p.propertyValue == '61') return p;} );
+        propertyDisplayValue = _.isUndefined(panels) && panels.length === 0 ? "" : panels[0].propertyDisplayValue;
```

Comment:
I don't get this condition, can this be true? If panels is undefined it can't have the panels.length

Reply:
Code is similar but can not be generic

Reply:
Changed the and operator to or operator.
Makes more sense